### PR TITLE
[34566] Remove references to todos

### PR DESCRIPTION
**Pull request by ChristopherCotnoir**

### CODE REVIEW

**Comment by ChristopherCotnoir** [comment]
Requires xtuple/xtuple#3569, associated with xtuple/qt-client#2229 and xtuple/connect#136.

File: resources/client/scripts/initMenu.js [hunk 1]
```diff
@@ -119,6 +119,9 @@ if (_desktopParent)
   {
     mainwindow.findChild("_todo").actionName = "crm.taskList";
     mainwindow.findChild("_todoCalendar").actionName = "crm.taskListCalendar";
+    mainwindow.findChild("_todo").label = "Task List";
+    mainwindow.findChild("_todoCalendar").label = "Task Calendar";
+    mainwindow.findChild("_dockMytodo").windowTitle = "Task";
   }
```
Comment (andersdd) [inline resources/client/scripts/initMenu.js]: Should these be run through potential translation?  `qsTr("Task List")` etc.

File: resources/database/misc/migratePriv.sql [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+BEGIN
+
+  IF (_normalizeversion(fetchMetricText('ServerVersion')))[1] >= 5 THEN
+    UPDATE priv
+       SET priv_name = replace(priv_name, 'Todo', 'Task'),
+           priv_descrip = replace(priv_descrip, 'To Do', 'Task')
+     WHERE priv_module = 'Desktop';
+  END IF;
+END
+$$ language plpgsql;
```
Comment (andersdd) [inline resources/database/misc/migratePriv.sql]: Why is a global privilege migration being done in the xtdesktop package?  Surely this should be done as part of core.  Have you checked qt-client and all packages for instances of the `Todo` privileges?

Comment (ChristopherCotnoir) [reply resources/database/misc/migratePriv.sql]: WHERE priv_module = 'Desktop' means only xtdesktop privs.

**Comment by ChristopherCotnoir** [comment]
Accidentally pushed branch to xtuple instead of my fork. Replaced by #113.

**Pull request closed without merging**